src/components/autoHeightImage/autoHeightImage.tsx

```diff
@@ -1,8 +1,8 @@
 import React, { useEffect, useState } from 'react';
-import { Image } from 'react-native';
+import { Image, TouchableOpacity } from 'react-native';
 import EStyleSheet from 'react-native-extended-stylesheet';
 import FastImage from 'react-native-fast-image';
-import { TouchableOpacity } from 'react-native-gesture-handler';
```

Comment:
@noumantahir RN touchableopacity had some issues that's why we were using another package, did you check if it doesn't break anything?

Reply:
at this point, I am undecided, because both show issues in different aspects... on android gesture-handler touchable stopped responding in recent updates, reported by melinda as well.... that is why shifted back to stock touchable.